test/e2e/util/hypershift_framework.go

```diff
@@ -253,6 +260,37 @@ func (h *hypershiftTest) createHostedCluster(opts *PlatformAgnosticOptions, plat
 		err = h.client.Create(h.ctx, serviceAccountSigningKeySecret)
 		g.Expect(err).NotTo(HaveOccurred(), "failed to create serviceAccountSigningKeySecret")
 
+		//create external oidc secret and configmap
+		if opts.AuthenticationConfig != nil && opts.ExtOIDCConfig != nil {
+			consoleCliSecret := &corev1.Secret{
```

Comment:
@heliubj18 Same nit here, how about renaming `consoleCliSecret` to be something like `consoleSecret` or `consoleClientSecret`?

Reply:
done